snmp/datadog_checks/snmp/data/profiles/juniper-ex.yaml

```diff
@@ -21,11 +21,31 @@ device:
   vendor: "juniper-networks"
 
 sysobjectid:
```

Comment:
I think it can help to add as comment what is the source documents that helped building this list. This can also help reviewers to QA/double check the list of sysobjectid.

Same comment for mx, srx.

Reply:
Ok!